lib/manageiq/providers/vmware/engine.rb

```diff
@@ -29,6 +29,9 @@ def self.init_loggers
 
           require "rbvmomi"
           RbVmomi.logger = $vim_log
+
+          require "vmware_web_service"
```

Comment:
NOTE this just autoload's some constants and requires `logging`: https://github.com/ManageIQ/vmware_web_service/blob/master/lib/vmware_web_service.rb